src/tir/usmp/utils.cc

```diff
@@ -168,6 +174,20 @@ Array<BufferInfo> CreateArrayBufferInfo(const Map<BufferInfo, Stmt>& buffer_info
   return ret;
 }
 
+void PrintConflicts(const Array<BufferInfo>& bi_arr) {
+  for (const auto& bi : bi_arr) {
```

Comment:
want to either log each conflict separately or wrap this function in a log level check? (or, is this function called?)

Reply:
Removed, it is not called.